.gitignore

```diff
@@ -13,7 +14,7 @@
 /.nb-gradle/private/
 
 # DMDirc development profile
-/etc/profile/*.*
+/etc/profile
```

Comment:
If you're ignoring /etc/profile then there's no point having the rules below it.

(I think that's fine, we've not updated the dev profile in forever and we can fiddle if necessary if we want to update it).